### PR TITLE
Update Teamwork.md with effective leadership tips section

### PR DESCRIPTION
**Pull request by Andyctct**
Adding a section to Teamwork.md that discusses what being an effective leader means, and provides some tips and sources regarding how we can try to be more effective leaders.

### CODE REVIEW

**Review by Jesse-Na — CHANGES_REQUESTED** [review]
Great contribution, you've provided a lot of excellent tips for people aspiring to be good leaders in a digestible and justified manner.

File: Topics/Teamwork.md [hunk 1]
```diff
@@ -83,3 +83,39 @@ Handling a non-responsive and unmotivated team member in software engineering ca
...
+
+## How to be an Effective Leader
+
+First, let’s start by asking “What does it mean to be an effective leader in the first place?”. Being an “effective leader” may mean delivering projects as fast as possible in one situation, but it may mean building a cohesive, trusting team culture in another. Every leader will find themselves in a different situation and what it means to be an effective leader may differ greatly in these various contexts. One of the first steps to being an effective leader is to acknowledge this, and then try to ascertain the best paths towards success, whatever that may look like in your environment. Thus, due to the variableness of what being an “effective leader” can mean, in this guide we simply note some approaches that we believe are widely applicable across a large range of environments. Also, note that we aim our guidance and examples at leadership in the software engineering space, but many of the concepts are naturally abstractable outside of this sphere as well.
```
[the remark this answers is not in the document]
Comment (Jesse-Na) [reply Topics/Teamwork.md]: Yeah I think your new line helps clarify things!

File: Topics/Teamwork.md [hunk 2]
```diff
@@ -86,31 +86,31 @@ However, if the team member continues to be non-responsive and unmotivated despi
...
...lace/leadership-principles), “Leaders do not believe their or their team’s body odor smells of perfume”. This means that leaders should not view themselves or their team as perfect and above the need for feedback (smelling like perfume in the example), but should instead stay grounded and open to improvement. Leaders should certainly be prepared to give feedback to the people they work with, but it is just as important to be open to any feedback that comes back. After all, we just discussed how hard it can be to pin down what an “effective leader” looks like in each situation, so why not let the people you lead help you? In fact, we believe that not only should you be open to feedback, but it is something that is worth actively seeking out as it is a leader’s responsibility to not only strive for constant team improvement, but also improvement of themselves and any mechanisms that they employ. One simple way to seek out feedback is to directly ask the people you are working with for it on a regular basis (such as during your weekly check-ins if you have them), but it is important to do so respectfully and in a manner such that the other person feels comfortable speaking their mind.
```
Comment (Jesse-Na) [inline Topics/Teamwork.md:93]: I think the sentence flows better, but I'll point out that you used "but" twice. See here: "but it is something that is worth actively seeking out as it is a leader’s responsibility to not only strive for constant team improvement, but also improvement of themselves and any mechanisms that they employ."

File: Topics/Teamwork.md [hunk 3]
```diff
@@ -83,3 +83,39 @@ Handling a non-responsive and unmotivated team member in software engineering ca
...
+
+**Be open to feedback, and seek it out**
+
+Although we just discussed how different being an effective leader can look from case to case, a staple in almost every effective leader is their openness to feedback and reiteration. As said in a leadership principle from a [famous company](https://www.amazon.jobs/content/en/our-workplace/leadership-principles), “Leaders do not believe their or their team’s body odor smells of perfume”. Leaders should certainly be prepared to give feedback to the people they work with, but it is just as important to be open to any feedback that comes back. After all, we just discussed how hard it can be to pin down what being an “effective leader” looks like in each situation, so why not let the people you lead help you? In fact, we believe that not only should you be open to feedback, but it is something that is worth actively seeking out as it is a leader’s responsibility to strive for constant improvement of the team, but also themselves and any mechanisms that they employ.
```
[the remark this answers is not in the document]
Comment (Jesse-Na) [reply Topics/Teamwork.md]: Ah, yeah the line you added helps me understand the expression better now. Nice!

**Review by Jesse-Na — APPROVED** [review]

File: Topics/Teamwork.md (hunk 2 above)
Comment (Andyctct) [reply Topics/Teamwork.md:93]: Thanks for pointing this out, I thought about this myself as well haha. I agree it is a little weird, but should still be clear enough to the reader so I will leave it in. I like to think of the second "but" as a leaf node extending from the first "but" ;).

**Pull request merged**